src/lib.rs

```diff
@@ -102,6 +130,13 @@ where
                     });
                 }
             }
+            OperationResult::Err(error) => {
+                return Err(Error::Operation {
+                    error: error,
```

Comment:
This line and the one below it can use the shorthand of only including the field name since it matches the value name. I think clippy will complain otherwise.

Reply:
fixed, cargo clippy is reporting clean